raiden/api/v1/encoding.py

```diff
@@ -327,3 +327,43 @@ class ConnectionsLeaveSchema(BaseSchema):
     class Meta:
         strict = True
         decoding_class = dict
+
+
+class EventTransferSchema(BaseSchema):
+    payment_network_identifier = fields.Str(missing=None)
+    token_network_identifier = fields.Str(missing=None)
+    channel_identifier = fields.Str(missing=None)
+    block_number = fields.Integer()
+    identifier = fields.Integer(missing=None)
+    event = fields.Str()
+
+
+class EventTransferSentFailedSchema(EventTransferSchema):
```

Comment:
IMO inheritance is overkill here, we are not inheriting any behavior, only a few attributes